File: Mashup1/recrutamento.py
nEventos = int(input())
i = 0
umOuMenosUm = []

valores = list(map(int, input('Digite os valores: ').split()))

while i < nEventos:
   num = int(input(), end=" ")
   if num <= 10:
      umOuMenosUm.append(num)
   i = i + 1

i = 0
x = 0
if umOuMenosUm[0] == -1:
   x = 1
if umOuMenosUm[len(umOuMenosUm)-1] == -1:
   x = x + 1
while i <= len(umOuMenosUm):
   if umOuMenosUm[i] == -1 and umOuMenosUm[i+1] == -1:
      x = x + 1
print(x)


   

File: Mashup1/recrutamento.c
#include <stdlib.h>
#include <stdio.h>

typedef struct celula {
   int dado;
   struct celula *prox;
} celula;

celula *enfileira(celula *f, int x){
   celula *novo = malloc(sizeof(celula));
   if(novo == NULL)
      return NULL;
   else{
      novo->prox = f->prox;
      f->prox = novo;
      f->dado = x;
      return novo;
   }
}


int main (){
   celula *umOuMenosUm = malloc(sizeof(celula));
   int nEventos;
   scanf("%d", &nEventos);

   for (int i = 0; i <= nEventos; i++)
   {
      int num;
      scanf("%d", &num);
      enfileira(umOuMenosUm, num);
   }
   
   while ()
   {
      /* code */
   }
   

   return 0;
}